### PR TITLE
Style Header Component

### LINKED ISSUE #30: Style Navigation Component (Header)
Style the Header Component to match the design document.

### PR DESCRIPTION
**Pull request by samuelsogbesan**
This PR integrates the styling for the Header component.

The outcome _(black was used here for illustration, but a CSS variable holding the colour white will be substituted in a later PR_)

_The navigation options here are purely illustrative, but will be merged via #125_
image

Expected result:

image

Closes #30

### CODE REVIEW

File: src/main/webapp/style.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+.navigation {
+  margin-left:auto;
+}
+
+.navigation > li {
+  padding: 0 2rem;
+  font-style: italic;
+  text-decoration: underline;
+}
```
Comment (holounic) [inline src/main/webapp/style.css:31]: nit: please add newline

**Review by holounic — APPROVED** [review]
LGTM

File: src/main/webapp/style.css (hunk 1 above)
Comment (samuelsogbesan) [reply src/main/webapp/style.css]: Added with commit b42d472

File: src/main/webapp/style.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
+a {
+  color:inherit;
```
Comment (Somae) [inline src/main/webapp/style.css:2]:
```suggestion
  color: inherit;
```

**Review by Somae — APPROVED** [review]
You can consider adding short comments that illustrate what the result / effect of the various CSS classes is summary (not every attribute on it's own, but the combined effect of applying it).

**Pull request merged**